libcontainer/cgroups/systemd/apply_systemd.go

```diff
@@ -151,8 +151,8 @@ func (m *Manager) Apply(pid int) error {
 		properties []systemdDbus.Property
 	)
 
-	if c.Slice != "" {
-		slice = c.Slice
+	if c.Parent != "" {
```

Comment:
Docker sets Cgroup Parent to be "docker" in daemon/execdriver/native/template/default_template_linux.go. This should be changed to "system.slice" in case of systemd cgroups.